src/main/java/org/parabot/launcher/data/Configuration.java

```diff
@@ -12,5 +12,5 @@
     public static final String CONFIG_NAME = "launcher";
     public static final String LAUNCHER_CONFIG_LOCATION = Directories.getSettingsPath() + File.separator + Configuration.CONFIG_NAME + ".json";
     public static final String CLIENT_LOCATION = Directories.getCachePath() + File.separator + BOT_TITLE + ".jar";
-    public static final String GET_BOT_VERSION = "http://bdn.parabot.org/api/v2/bot/version";
+    public static final String GET_BOT_VERSION = "http://v3.bdn.parabot.org/api/bot/list/client?limit=1";
```

Comment:
I'm saying, that's using the same variable; https://github.com/Parabot/Parabot/blob/development/src/main/java/org/parabot/core/Configuration.java#L18